src/base/memory_hash.c

```diff
@@ -1160,7 +1162,11 @@ mht_dump (THREAD_ENTRY * thread_p, FILE * out_fp, const MHT_TABLE * ht, const in
 	   "HTABLE NAME = %s, SIZE = %d, REHASH_AT = %d,\n" "NENTRIES = %d, NPREALLOC = %d, NCOLLISIONS = %d\n\n",
 	   ht->name, ht->size, ht->rehash_at, ht->nentries, ht->nprealloc_entries, ht->ncollisions);
 
-  if (print_id_opt)
+  if (print_id_opt == -1)
+    {
+      /* noting to do */
+    }
+  else if (print_id_opt)
```

Comment:
"else" clause mean print_id_opt is not -1, it includes 0 and any positive value but also includes -2, -100, and any negative values.

Reply:
The print_id_opt variable was previously used as a boolean. I've added'-1' to print nothing. 
I think there is a no problem Because '0' is excluded.